src/client/actions/edit/articleActions.js

```diff
@@ -0,0 +1,211 @@
+import { clone, cloneDeep, debounce, set } from 'lodash'
```

Comment:
IMO this is exactly the right path -- take an actions file and when it gets too large group it under an appropriate subfolder 💯. The same process typically happens at the reducer level as well, so that things maintain a 1:1 pairity.